extensions/amp-ad-exit/0.1/config.js

```diff
@@ -121,6 +131,13 @@ function assertTarget(name, target, config) {
       user().assert(
           pattern.test(variable), '\'%s\' must match the pattern \'%s\'',
           variable, pattern);
+      const vendor = variable.substr(1);
```

Comment:
Apparently so - caught by unit test. Glad to see they're working :)
"Error: Unknown vendor: foo​​​"